superset/utils/decorators.py

```diff
@@ -34,6 +34,10 @@
 logger = logging.getLogger(__name__)
 
 
+def is_dashboard_request(kwargs: Any) -> bool:
+    return kwargs.get("dashboard_id_or_slug") is not None
```

Comment:
doesn't seem robust, it it possible to validate via uri path or just pass a param ?

Reply:
after introduce this skip, `dashboard_id_or_slug` is not needed in decorator function any more. this check is removed.